install/kubernetes/helm/istio/values.yaml

```diff
@@ -136,19 +151,21 @@ global:
     # Resources for the sidecar.
     resources:
       requests:
-        cpu: 10m
-      #  memory: 128Mi
-      # limits:
-      #   cpu: 100m
-      #   memory: 128Mi
+        cpu: 100m
+        memory: 128Mi
+      limits:
+        cpu: 2000m
+        memory: 128Mi
```

Comment:
wow big cpu bump for pilot.  cc @jnativio

Reply:
that is a limit, not request.

Reply:
I understand the requests/limits values above are for sidecars. We don't have limits defined by default currently. Nor memory requests in 1.1 branch. Maybe a lower value for memory requests should be more adequate (it is set as 30Mi in the master branch).

Reply:
yes, you are right it is for sidecar.  I'm a little concerned on the big memory requests (and also cpu req) for initial sidecar.  limits seems okay.  also, it is odd we have the same number for memory request and memory limit.

Reply:
We run a 700 pod cluster, and in our tests from istio 1.0.6 to 1.1.0-rc0 our cluster cost would go from 100k to 202k if a limit on the CPU for the sidecar is put to `2000m`.  This requires every pod running to take an additional 2 CPU cores.  Remember that a limit requires kubernetes to ensure that amount of CPU is always available for the sidecar as well, which makes it a dedicated resource and VERY expensive.

Reply:
Actually, the scheduler takes in consideration the requests value only. The limits value is used to curb the CPU/memory usage by the container.

Reply:
Yeah, @jnativio I'm going to go ahead and apologize profusely, this was my misunderstanding as when I went back and looked closer the node went un-scheduable for a completely unrelated reason.  Sorry. Sorry. Sorry.